client/cherami/outputhostconnection.go

```diff
@@ -161,7 +159,7 @@ func (conn *outputHostConnection) close() {
 		}
```

Comment:
Not related to this change but using atomic.LoadInt32 here is unnecessary.  There is already a lock on entering close.